packages/cli/src/utils/platform.ts

```diff
@@ -119,19 +119,71 @@ export const platformRequestOrThrow = async (params: {
   return json
 }
 
-export const dispatchToSubCommand = async (commands: Commands, argv: string[]) => {
+export const dispatchToSubCommand = async (commands: Commands, argv: string[], help?: (error?: string) => string) => {
   const commandName = argv[0]
-  // Placeholder for now
-  const helpPlaceholder = 'Coming soon: help output for this command.'
-  if (!commandName) return helpPlaceholder
+  if (['-h', '--help'].includes(commandName)) return help?.() ?? ''
```

Comment:
I am still bouncing back and forth whether `help` method should be passed as an argument to `dispatchToSubCommand`

Benefits:
- Wraps the logic for all platform commands avoiding repeating logic and error checking on the command level

Con:
- This pattern does not work for commands without sub commands. These commands would anyways need to add logic to their command definition.  As a result we would have two different ways of handling help calls